Tests/SentryTests/Integrations/Breadcrumbs/SentryBreadcrumbTrackerTests.swift

```diff
@@ -230,6 +230,82 @@ class SentryBreadcrumbTrackerTests: XCTestCase {
         XCTAssertEqual(payload["message"] as? String, "methodPressed:")
     }
     
+    func testTouchBreadcrumb_DontReportAccessibilityIdentifier() throws {
+        let scope = Scope()
+        let client = TestClient(options: Options())
+        let hub = TestHub(client: client, andScope: scope)
+        SentrySDK.setCurrentHub(hub)
+        
+        let swizzlingWrapper = TestSentrySwizzleWrapper()
+        SentryDependencyContainer.sharedInstance().swizzleWrapper = swizzlingWrapper
+        
+        let tracker = SentryBreadcrumbTracker(reportAccessibilityIdentifier: false)
+        tracker.start(with: delegate)
+        tracker.startSwizzle()
+        
+        let button = UIButton()
+        button.accessibilityIdentifier = "TestAccessibilityIdentifier"
+        
+        swizzlingWrapper.execute(action: "methodPressed:", target: self, sender: button, event: TestEvent(touchedView: button))
+        
+        guard let crumb = delegate.addCrumbInvocations.invocations.first(where: { $0.category == "touch" }) else {
```

Comment:
`l`: We always use XCTUnwrap in the tests. I would prefer sticking to this approach, and the syntax is shorter.

Reply:
Ctrl+C Ctrl+V 😅
I just changed the important parts.